lerna.json

```diff
@@ -0,0 +1,14 @@
+{
+  "npmClient": "npm",
```

Comment:
lerna er ett verktøy for monorepoer, hvorfor er det tatt inn her?

Reply:
Sant det, men fungerer veldig greit for å håndtere publisering av pakker. 
Fant ingen god måte for `semantic-release` å publisere til to registre samtidig, så tenker lerna kan fungere greit som erstatning selv om den er litt "heavy" for bruken.